File: jvm2/src/Model/Neutral/Units/Worker.kt
package Model.Neutral.Units

import Model.Abstract.Unit

abstract class Worker() : Unit() {
    override var occ = true
    override var traverse = false
    override var minCost = 20
    override var gasCost = 10
    override var nrgCost = 0
    override var health = 20
    override var commands = mutableListOf<String>("Gather resources", "Abort")
}

File: dotnet/src/Model/Neutral/Units/Worker.cs
using System;
using System.Collections.Generic;

namespace PlanetCS1 {
    public abstract class Worker : Unit {
        public Worker() {
            this.occ = true;
            this.traverse = false;
            this.minCost = 20;
            this.gasCost = 10;
            this.nrgCost = 10;
            this.health = 20;
            this.commands = new List<String>(){"Gather resources", "Abort"};
        }
    }
}